src/Generation/GapicClientExamplesGenerator.php

```diff
@@ -93,7 +93,15 @@ private function initCallVars(MethodDetails $method): array
 
                     if (!$f->useResourceTestValue) {
                         // Use a default example value if no values are specified.
-                        $initCode = AST::assign($var, $f->exampleValue($this->ctx));
+                        if ($f->isOneOf) {
+                            $initCode = AST::assign(AST::var(Helpers::toCamelCase($f->containingMessage->getOneofDecl()[$f->oneOfIndex]->getName())),
+                                AST::call(
+                                    AST::new($this->ctx->type($f->toOneofWrapperType($method->serviceDetails->namespace)))(),
+                                    AST::method("set" . Helpers::toUpperCamelCase($f->camelName)))($f->exampleValue($this->ctx))
+                            );
+                        } else {
+                            $initCode = AST::assign($var, $f->exampleValue($this->ctx));
+                        }
```

Comment:
Nit: Is this just a code style thing to move the "default case" into the `else` rather than have it always execute prior to the "special case" `if ($f->isOneOf)`?

Reply:
Yes. Similar patterns elsewhere made code parsing easier for me, but I don't have a strong preference.

Reply:
Ok! That's fine with me, especially for consistency.

My personal preference is to keep "basic case" code executing before the conditionals for the "special" cases, if that code has no undesirable side effects, like this which is just assigning a value to a variable. That keeps the main execution of the code left justified and the special cases indented.